docs/tutorials/sinks/1_basic_sink.md

```diff
@@ -40,12 +57,14 @@ pub struct BasicConfig {
 }
 ```
 
-Note the `configurable_component` option. This is used by Vector to generate documentation from the struct.
-To do this, doc comments must be included above the struct - Vector won't compile if they aren't.
+Note the `configurable_component` attribute. This is used by Vector to generate
+documentation from the struct. To do this, doc comments must be included above
+the struct - Vector won't compile if they aren't.
 
-We also include a single member in our struct - `acknowledgemets`. Every sink needs to be able to control how
-it handles acknowledgements - do connected sources need to wait for this sink to send the message before
-acknowledging that they have received the events. We will make this a configurable option.
+We also include a single member in our struct - `acknowledgemets`. Every sink
+needs to be able to control how it handles acknowledgements - do connected
+sources need to wait for this sink to send the message before acknowledging
+that they have received the events. We will make this a configurable option.
```

Comment:
I'm no expert on e2e acks but this wording is confusing to me.
The below is an attempt, but I'm not sure how accurate it is.
The "to send the message" is ambiguous, and the "they have received the events", who does the "they" refer to?

```suggestion
needs to be able to control how it handles acknowledgements. This struct
configures if and how connected sources need to wait for this sink to send the
message before acknowledging that they have received the events.
```

Reply:
It's quite a complicated subject. I'll defer to Vectors docs to explain it better.